### PR TITLE
fix(RESTPatchAPIChannelJSONBody): add missing `applied_tags` field

### PR DESCRIPTION
**Pull request by suneettipirneni**
**Please describe the changes this PR makes and why it should be merged:**

**If applicable, please reference Discord API Docs PRs or commits that influenced this PR:**

### CODE REVIEW

File: deno/rest/v10/channel.ts [hunk 1]
```diff
@@ -181,6 +181,11 @@ export interface RESTPatchAPIChannelJSONBody {
 	 * Channel types: forum
 	 */
 	default_forum_layout?: ForumLayoutType | undefined;
+	/**
+	 * the IDs of the set of tags that have been applied to a thread in a
```
Comment (vladfrangu) [inline deno/rest/v10/channel.ts:185]: Start with capitalized word

**Review by vladfrangu — CHANGES_REQUESTED** [review]

**Review by vladfrangu — DISMISSED** [review]

File: rest/v9/channel.ts [hunk 2]
```diff
@@ -181,6 +181,13 @@ export interface RESTPatchAPIChannelJSONBody {
 	 * Channel types: forum
 	 */
 	default_forum_layout?: ForumLayoutType | undefined;
+	/**
+	 * The IDs of the set of tags that have been applied to a thread in a
+	 * GUILD_FORUM or a GUILD_MEDIA channel; limited to 5
```
Comment (almeidx) [inline rest/v9/channel.ts:186]:
```suggestion
	 * The ids of the set of tags that have been applied to a thread-only channel; limited to 5
```

File: rest/v10/channel.ts [hunk 3]
```diff
@@ -181,6 +181,13 @@ export interface RESTPatchAPIChannelJSONBody {
 	 * Channel types: forum
 	 */
 	default_forum_layout?: ForumLayoutType | undefined;
+	/**
+	 * The IDs of the set of tags that have been applied to a thread in a
+	 * GUILD_FORUM or a GUILD_MEDIA channel; limited to 5
```
Comment (almeidx) [inline rest/v10/channel.ts:186]:
```suggestion
	 * The ids of the set of tags that have been applied to a thread-only channel; limited to 5
```

File: deno/rest/v9/channel.ts [hunk 4]
```diff
@@ -181,6 +181,13 @@ export interface RESTPatchAPIChannelJSONBody {
 	 * Channel types: forum
 	 */
 	default_forum_layout?: ForumLayoutType | undefined;
+	/**
+	 * The IDs of the set of tags that have been applied to a thread in a
+	 * GUILD_FORUM or a GUILD_MEDIA channel; limited to 5
```
Comment (almeidx) [inline deno/rest/v9/channel.ts:186]:
```suggestion
	 * The ids of the set of tags that have been applied to a thread-only channel; limited to 5
```

File: deno/rest/v10/channel.ts [hunk 5]
```diff
@@ -181,6 +181,13 @@ export interface RESTPatchAPIChannelJSONBody {
 	 * Channel types: forum
 	 */
 	default_forum_layout?: ForumLayoutType | undefined;
+	/**
+	 * The IDs of the set of tags that have been applied to a thread in a
+	 * GUILD_FORUM or a GUILD_MEDIA channel; limited to 5
```
Comment (almeidx) [inline deno/rest/v10/channel.ts:186]:
```suggestion
	 * The ids of the set of tags that have been applied to a thread-only channel; limited to 5
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Review by almeidx — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

**Review by HeadTriXz — APPROVED** [review]
LGTM

**Review by Jiralite — APPROVED** [review]

**Pull request merged**